src/util/modeControl.js

```diff
@@ -0,0 +1,84 @@
+L.Control.MinimalMode = L.Control.extend({
+
+    options: {
+      position: 'topleft',
+      minimalMode: false
+    },
+  
+    initialize: function(layersControl, options) {
+      L.Util.setOptions(this, options);
+      this._layersControl = layersControl;
+    },
+  
+    onAdd: function(map) {
+      this._map = map;
+      this._map._minimalMode = this.options.minimalMode;
+      this._modeBtnContainer = L.DomUtil.create('div', 'leaflet-control-mode leaflet-bar leaflet-control');
+      this._defaultMode = this._createButton('Default mode', 'fas fa-map-marker-alt', this.loadDefaultMode);
+      this._minimalMode = this._createButton('Minimal mode', 'far fa-dot-circle', this.loadMinimalMode);
```

Comment:
Maybe this could be something more descriptive, like "Default markers" and "Dot markers"? Or "Minimal markers"?

Reply:
Do you mean in L17 and L18? If so for clarity I could change it to defaultModeBtn? I think it will be misleading to add 'markers' to the button's name.

Reply:
Yes, I mean for the text that is in the pop-up, so it is descriptive to users. Changing the name to `defaultModeBtn` would be good!

Reply:
Ah, that one! That's a great idea!